### PR TITLE
Feature/nickname quiddity

### PR DESCRIPTION
**Pull request by francoisph**

### CODE REVIEW

**Comment by francoisph** [comment]
@ubald code updated

File: client/src/model/Quiddity.js [hunk 1]
```diff
@@ -139,6 +143,11 @@ const Quiddity = ScenicModel.extend( {
             // since it's the same object (the previous value ends up referencing the current one in backbone)
             this.trigger( 'change:userData' );
             this.trigger( 'change:userData:' + path );
+
+            // update the name of the quiddity 
+            if( path == 'nickname' ) {
+                this.set('name',  value );
```
Comment (ubald) [inline client/src/model/Quiddity.js]: Do this instead
```js
this.trigger( 'change:name' );
```

File: client/src/view/scenic/inspector/information/EditQuiddity.html [hunk 2]
```diff
@@ -1,7 +1,25 @@
+<div class="edit">
+<% if ( !editName ) { %>
+	<div class="title">Edit <%=name%>
```
Comment (ubald) [inline client/src/view/scenic/inspector/information/EditQuiddity.html]: i18n

**Comment by francoisph** [comment]
@ubald code updated

File: client/src/model/Quiddity.js [hunk 3]
```diff
@@ -139,6 +143,7 @@ const Quiddity = ScenicModel.extend( {
             // since it's the same object (the previous value ends up referencing the current one in backbone)
             this.trigger( 'change:userData' );
             this.trigger( 'change:userData:' + path );
+            this.trigger( 'change:name' );
```
Comment (ubald) [inline client/src/model/Quiddity.js]:
```js
if ( path == 'nickname' ) ...
```

**Pull request merged**